### PR TITLE
added run, crouch, bleed status icons for RetroDays sees_player_retro mod

### PR DESCRIPTION
**Pull request by dranitski**
#### Summary
Interface "Status icons for sees_player_retro mod"

#### Purpose of change
See updated version in https://github.com/CleverRaven/Cataclysm-DDA/pull/51910
~~RetroDays sees_player_retro mod adds alert icon for enemies, but lacks run, crouch, bleed icons that i've added.~~

#### Describe the solution
~~Added the icons, updated the configs and mod description.~~

#### Testing
Works fine
2021-08-29_19-53 Cropped
2021-08-29_19-54 Cropped
2021-08-29_19-55 Cropped

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
How does it look on mutants with larger sprites?

**Comment by dseguin** [comment]
Looks like the reversible sprites might be an issue when facing left:
run_backwards
crouch_backwards

**Comment by dranitski** [comment]
> How does it look on mutants with larger sprites?

I sure RetroDays don’t have any player sprites bigger than 10x10 so that may not be an issue. https://github.com/CleverRaven/Cataclysm-DDA/blob/master/gfx/RetroDaysTileset/tile_config.json

**Comment by dranitski** [comment]
> Looks like the reversible sprites might be an issue when facing left:
> run_backwards
> crouch_backwards

Maybe `, "rotates": false` in tile config will solve that? Will take a look

**Comment by dranitski** [comment]
okay, seems like `rotate: true` is not connected with tile flipping. I've changed icons from `C` and `R` letters to `plus` signs  to address that issue

image

File: data/mods/sees_player_retro/modinfo.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   {
     "type": "MOD_INFO",
     "id": "sees_player_retro",
-    "name": "sees-player icon, retrodays",
+    "name": "sees-player, run, crouch, bleed icons, retrodays",
     "authors": [ "esotericist" ],
-    "description": "Adds indicator icon if a creature sees the player.  Designed for the retrodays tileset.",
+    "description": "Adds indicator icon if a creature sees the player. Adds status icons for run, crouch, bleeding on top of player tile. Designed for the retrodays tileset.",
```
Comment (kevingranade) [inline data/mods/sees_player_retro/modinfo.json:7]:
```suggestion
    "description": "Adds indicator icon if a creature sees the player.  Adds status icons for run, crouch, bleeding on top of player tile.  Designed for the retrodays tileset.",
```

**Comment by anothersimulacrum** [comment]
@dranitski in the 20x20 tiles:
https://github.com/CleverRaven/Cataclysm-DDA/blob/a025ee9f3d2f6a4fcd72cc2f3162b72f84b1c935/gfx/RetroDaysTileset/tile_config.json#L8655-L8658

**Comment by dranitski** [comment]
updated version is in https://github.com/CleverRaven/Cataclysm-DDA/pull/51910

**Pull request merged**